sig-security/security-audit-2021/RFP.md

```diff
@@ -0,0 +1,121 @@
+# Request for Proposal
+
+## Kubernetes Third-Party Security Audit
+
+The Kubernetes Third-Party Audit Working Group (working group, henceforth) is soliciting proposals from Information Security vendors for a comprehensive security audit of the Kubernetes Project.
```

Comment:
Should probably change all these references to "Kubernetes SIG Security Third-Party Audit Subproject", for completeness' sake.

Reply:
done.